com.endlessm.DragonsApprentice.json

```diff
@@ -0,0 +1,58 @@
+{
+    "app-id": "com.endlessm.DragonsApprentice",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "com.endlessm.DragonsApprentice.sh",
+    "separate-locales": false,
+    "build-options" : {
+        "no-debuginfo": true,
+        "strip": true
+    },
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri"
+    ],
+    "modules": [
+        {
+            "name": "DragonsApprentice",
+            "buildsystem": "simple",
+            "only-arches": ["x86_64"],
+            "build-commands": [
+                "install -Dm644 com.endlessm.DragonsApprentice.desktop /app/share/applications/com.endlessm.DragonsApprentice.desktop",
+                "install -Dm644 com.endlessm.DragonsApprentice.appdata.xml /app/share/appdata/com.endlessm.DragonsApprentice.appdata.xml",
+                "install -Dm644 com.endlessm.DragonsApprentice.png /app/share/icons/hicolor/128x128/apps/com.endlessm.DragonsApprentice.png",
+                "install -D com.endlessm.DragonsApprentice.sh /app/bin/com.endlessm.DragonsApprentice.sh",
+                "unzip DragonsApprentice.zip",
+                "install DragonsApprentice.x86_64 /app/bin/DragonsApprentice.x86_64",
+                "mv DragonsApprentice_Data /app/bin/"
+            ],
+            "sources" : [
+                {
+                    "type": "file",
+                    "path": "https://www.dropbox.com/s/id2zldbkidrhynm/DragonsApprentice.zip"
+                },
+                {
+                    "type": "file",
+                    "path": "https://github.com/JasonFPL/DragonsApprentice/blob/master/com.endlessm.DragonsApprentice.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "https://github.com/JasonFPL/DragonsApprentice/blob/master/com.endlessm.DragonsApprentice.appdata.xml"
```

Comment:
## URLs

```
  <url type="homepage">https://endlessos.com</url>
  <url type="bugtracker">https://community.endlessos.com/</url>
```
The URL should point to the project website, not general organisation.

## Description

```
  <description>
    <p>A game where you hack things using spells.</p>
  </description>
```
It is a good place to put basic info about license (i.e. freeware, freemium) and development state (i.e. early access, beta, alpha, pre-alpha).

## Inconsistent indent

```
  <releases>
		<release version="1.0" date="2018-09-09"/>
  </releases>
```

Please replaces tabs with spaces here.
```
  <releases>
    <release version="1.0" date="2018-09-09"/>
  </releases>
```

## Screenshots

```
  <screenshots>
    <screenshot type="default">
      <image>DA1.jpg</image>
      <caption>Solve puzzles</caption>
    </screenshot>
    <screenshot>
      <image>DA2.jpg</image>
      <caption>Epic adventure</caption>
    </screenshot>
    <screenshot>
      <image>DA3.jpg</image>
      <caption>Epic adventure</caption>
    </screenshot>
```

You should use URLs here. For example:
```
  <screenshots>
    <screenshot type="default">
      <image>https://raw.githubusercontent.com/JasonFPL/DragonsApprentice/master/DA1.jpg</image>
      <caption>Solve puzzles</caption>
    </screenshot>
    <screenshot>
      <image>https://raw.githubusercontent.com/JasonFPL/DragonsApprentice/master/DA2.jpg</image>
      <caption>Epic adventure</caption>
    </screenshot>
    <screenshot>
      <image>https://raw.githubusercontent.com/JasonFPL/DragonsApprentice/master/DA3.jpg</image>
      <caption>Epic adventure</caption>
    </screenshot>
```

## Inconsistent indent (2)

Replace this:
```
    <content_rating type="oars-1.1">
```
with this:
```
  <content_rating type="oars-1.1">
```

## Email

```
  <update_contact>nick_at_endlessm.com</update_contact>
```
Are you sure that this email is valid? I mean the `nick` part, not `_at_`.

## Launchable tag

You have to specify the `launchable` tag for `desktop-id`.
https://github.com/flathub/flathub/wiki/AppData-Guidelines#launchable
https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#qsr-app-launchable-info
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-launchable

```
  <launchable type="desktop-id">com.endlessm.DragonsApprentice.desktop</launchable>
```

Reply:
I can guarantee that email is no longer valid!